binder/postBuild

```diff
@@ -1 +1,3 @@
-pip install git+https://github.com/AeroMAPS/AeroMAPS.git
\ No newline at end of file
+pip install poetry
```

Comment:
I would fix the poetry version like `pip install poetry==1.8.3` for more stability.